tests/document.js

```diff
@@ -0,0 +1,212 @@
+      var year = today.getFullYear();
+      var date =  year + '-0' + month + '-' + day;
+
+      request
+          .get('/api/documents?date=' + date)
+          .set('x-access-token', token)
+          .set('limit', 2)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(2);
+            done();
+          });
+    });
+
+    it('returns message if no documents are found', function (done) {
+      request
+          .get('/api/documents?date=2016-09-13')
+          .set('x-access-token', token)
+          .expect({message: 'No documents for this date'}, done);
+    });
+
+    it('gets documents published by the same role level(admin)', function (done) {
+      request
+          .get('/api/documents?role=admin')
+          .set('x-access-token', token)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(4);
+            done();
+          });
+    });
+
+    it('gets documents published by the same role level(user)', function (done) {
+      request
+          .get('/api/documents?role=user')
+          .set('x-access-token', token)
+          .end(function (err, res) {
+            expect(res.status).to.equal(200);
+            expect(res.body).to.exist;
+            expect(Array.isArray(res.body)).to.equal(true);
+            expect(res.body).to.have.length(3);
+            done();
+          });
+    });
+  });
+
+  describe('/documents/:id test suite', function () {
+    it('returns specific document', function (done) {
+      request
+        .get('/api/documents/' + documentId)
+        .set('x-access-token', token)
+        .end(function (err, res) {
+          expect(res.status).to.equal(200);
+          expect(res.body).to.exist;
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.keys('_id', 'title', 'content', 'role', 'modifiedAt', 'ownerId', 'accessLevel', 'createdAt', '__v');
+          done();
+        });
+    });
+
+    it('returns message if document not found', function (done) {
+      request
+        .get('/api/documents/57d11f44b0a303c1186273cf')
+        .set('x-access-token', token)
+        .end(function (err, res) {
+          expect(res.status).to.equal(409);
+          expect(res.body.message).to.equal('Document not found');
+          done();
+        });
+    });
+
+    it('updates a document', function (done) {
+      request
+        .put('/api/documents/' + documentId)
+        .set('x-access-token', token)
+        .send({
+          title: 'Updated Doc',
+          content: 'Whatever',
+          accessLevel: 'private'
+        })
+        .expect(200)
+        .expect({message: 'Document has been updated'}, done);
+    });
+
```

Comment:
Add a test to check whether a person who does not own the document can edit or delete it.